ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -180,6 +180,12 @@
           + "the active is dead and until a new active is elected. Default value is \"false\". ";
   public static final boolean KSQL_QUERY_PULL_ENABLE_STANDBY_READS_DEFAULT = false;
 
+  public static final String KSQL_QUERY_PULL_STALE_READS_LAG_MAX_OFFSETS_CONFIG =
```

Comment:
This can be overridden at the query level with something like `SET 'ksql.query.pull.stale.reads.lag.max.offsets' = '100';` . This same param can even be set in a curl request.

I had originally added a header, but this existing mechanism seemed like a preferable way to go.  When you set it in the shell, it gets sent up throughout your session.